test/test_autograd.py

```diff
@@ -1432,6 +1432,349 @@ def backward(self, dy):
             y = MyOp.apply(x)
         self.assertFalse(y.requires_grad)
 
+    def _is_inference_tensor(self, tensor):
```

Comment:
Nit: it's probably worth splitting these tests into a new TestCase class(even a new file) since they're not really testing against autograd system.

Reply:
Moved to separate TestClass for now